e2e_tests/tests/command/test_run.py

```diff
@@ -296,6 +298,46 @@ def test_shm_size_command(
         _run_and_verify_exit_code_zero(cmd)
 
 
+@pytest.mark.slow
+@pytest.mark.e2e_cpu
+def test_docker_flags_command(tmp_path: Path) -> None:
+    _run_cmd_with_config_expecting_failure(
+        "exit 0",
+        "invalid docker flags: unknown flag: --a",
+        {"environment": {"docker_flags": ["--a"]}},
+    )
+
+    # Verify CPU shares config is applied.
+    cpu_shares = 17
+    with cmd.interactive_command(
+        "command",
+        "run",
+        "--config",
+        f"environment.docker_flags=['--cpu-shares={cpu_shares}']",
+        "echo hello world; echo hello world; sleep infinity",
+    ) as command:
+        assert command.task_id is not None
+        for line in command.stdout:
+            if "hello world" in line:
+                break
+
+        parent = cmd.get_command(command.task_id)["container"]["parent"]
+        client = docker.from_env()
+        running_containers = client.containers.list()
+        for i in running_containers:
+            if i.labels.get("ai.determined.container.parent") == parent:
```

Comment:
I have a pr floating somewhere where I removed parent label (in favor or "description" or something less actor-flavored). Just noting one of us with conflict the other on merge.